kong/kong_test.go

```diff
@@ -151,7 +156,13 @@ func runWhenEnterprise(t *testing.T, semverRange string, rbacRequired bool) {
 
 	r := res["configuration"].(map[string]interface{})["rbac"].(string)
 
-	if rbacRequired && r != "on" {
+	if required.rbac && r != "on" {
+		t.Skip()
```

Comment:
optional: since this logic is becoming more complicated, add an explanation like

```go
t.Log("no RBAC support in the Kong instance under test - skipping this test case")
```

before instances of `t.Skip()`